code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -0,0 +1,96 @@
+/obj/item/living_heart
+	name = "Living Heart"
+	desc = "Link to the worlds beyond."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "living_heart"
+	var/datum/mind/target
+
+/obj/item/living_heart/attack_self(mob/user)
+	. = ..()
+	if(!isecultist(user))
+		return
+	if(!target)
+		return
+	var/dist = get_dist(user,target)
+
+	switch(dist)
+		if(0 to 5)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is near you</span>")
+		if(6 to 15)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is somewhere in your vicinty</span>")
+		if(16 to 64)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is far away from you</span>")
+		else
+			to_chat(user,"<span class='warning'>[target.current.real_name] is beyond our reach</span>")
+
+	if(target.current.stat == DEAD)
+		to_chat(user,"<span class='warning'>[target.current.real_name] is dead. Bring him onto the transmutation rune!</span>")
```

Comment:
```suggestion
		to_chat(user,"<span class='warning'>[target.current.real_name] is dead. Bring them onto a transmutation rune!</span>")
```
you could also use that special pronoun proc thingie

Reply:
if he is gibbed/cremated/otherwise delted look at line 14. since it will point to a null, the function will just return.